client/rest-high-level/src/test/java/org/elasticsearch/client/BulkRequestWithGlobalParametersIT.java

```diff
@@ -149,34 +149,16 @@ public void testGlobalType() throws IOException {
         bulk(request);
 
         Iterable<SearchHit> hits = searchAll("index");
-        assertThat(hits, everyItem(hasType("global_type")));
-    }
-
-    @SuppressWarnings("unchecked")
-    public void testTypeGlobalAndPerRequest() throws IOException {
```

Comment:
It might be nice to leave this in (with `expectWarnings` calls), as I don't see this functionality being tested elsewhere.

Reply:
@jtibshirani  Thanks for the suggestion, Julie. But this test will not work anymore, as creating an IndexRequest without a type like `new IndexRequest("index2").id("2") ` will assign a type of `_doc` to it. While the test here checks that that that the result type  should be `"global_type"`.

Reply:
I am going to temporary disable this test with `AwaitsFix`, and let @markharwood  address this test through his bulk API change.

Reply:
Oh, I didn't realize this was actually broken. I now see it is also broken for update and delete. I would also be okay adding `AwaitsFix`, and we can work to fix the issue as soon as possible in another PR.